test/e2e/benchmark/throughput.go

```diff
@@ -47,7 +47,7 @@ func TwoNodeSimple(logger *log.Logger) error {
 		LocalTracingType:   "local",
 		PushTrace:          false,
 		DownloadTraces:     false,
-		TestDuration:       30 * time.Second,
+		TestDuration:       2 * time.Minute,
```

Comment:
The test duration needs to be extended to ensure enough transactions are produced.